platformio.ini

```diff
@@ -12,7 +12,8 @@
 framework = arduino
 platform = https://github.com/platformio/platform-espressif32.git
 build_flags =
-	-std=gnu++17
+	-std=gnu++2a
+	-frtti
```

Comment:
Added this so it builds on Windows as it seems Arduino framework was not compiled with this flag set so it errors out on dynamic_cast.